src/test/sourcemap_test.ts

```diff
@@ -87,7 +87,7 @@ suite('Bundler', () => {
 
   suite('Sourcemaps', () => {
 
-    test('inline', async() => {
+    test('inline', async () => {
```

Comment:
while you're here, can the test names better describe what they're testing for, taking into account how mocha will print them, ie "Sourcemaps inline - passed"